.github/CONTRIBUTING.md

```diff
@@ -1,27 +1,86 @@
-如果你想帮助 **ECharts** 的话，请首先阅读指导原则：
+# Contributing
 
-- 如果你希望提 bug：
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
 
-  - 清楚地描述问题，注意：**别人是否能通过你的描述理解这是什么问题**。
-  - 说明版本（ECharts 版本，浏览器版本，设备、操作系统版本等）
-  - 最好提供完整的ECharts option，截图或线上实例（可以使用JSFiddle/JSBin/Codepen）。
+Please check out the [Apache Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
 
-- 如果你想问问题：
+## Issues
 
-  - 首先是否在这些文档中寻找过问题的答案：[option文档](http://echarts.baidu.com/option.html)，[API文档](http://echarts.baidu.com/api.html)，[教程](http://echarts.baidu.com/tutorial.html)
-  - 简单的问题，可以在QQ群中求助（群号：465958031）
+When opening new issues, please use the [echarts issue helper](https://ecomfe.github.io/echarts-issue-helper/), opening issues in any other way will cause our robot to close them automatically
 
-- 如何取得能运行的 `ECharts option` 
+And before you do so, please search for similar questions in [issues list](https://github.com/apache/incubator-echarts/issues?utf8=%E2%9C%93&q=is%3Aissue). If a closed issue have new problems in your environment, please create a new issue and reference the previous one.
```

Comment:
This might be better wording:
```
And before doing so, please search for similar questions in our [issues list](https://github.com/apache/incubator-echarts/issues?utf8=%E2%9C%93&q=is%3Aissue). If you are able to reproduce an issue found in a closed issue, please create a new issue and reference the closed one.
```